lib/phoenix/digester.ex

```diff
@@ -294,6 +298,36 @@ defmodule Phoenix.Digester do
     end
   end
 
+  @doc """
+  Deletes compiled/compressed asset files, including the cache manifest.
+
+  ## Arguments
+
+    * `path` - The path where the compiled/compressed files are saved
+
+  """
+  @spec clean_all(String.t()) :: :ok | {:error, :invalid_path}
+  def clean_all(path) do
+    if File.exists?(path) do
+      %{"digests" => digests} = load_manifest(path)
+
+      grouped_digests = group_by_logical_path(digests)
+      logical_paths = Map.keys(grouped_digests)
+
+      files =
+        for {_, versions} <- grouped_digests,
+            file <- Enum.map(versions, fn {path, _attrs} -> path end),
+            do: file
+
+      remove_files(files, path)
+      remove_compressed_files(logical_paths, path)
+      remove_manifest(path)
+      :ok
+    else
+      {:error, :invalid_path}
```

Comment:
I think we should just return `:ok`. If there is no manifest, it is because nothing was generated, which means `--all` was removed?

Reply:
Ah, good point! :+1:

Reply:
Okay, so I made `clean_all/1` a no-op, but I did so by checking map_size(digests) > 0 because we always get a map back from load_manifest.  Wdyt? If it's too noisy I can re-organize.

Reply:
Ah, or we could just `rm` and not check for existence :) 🙈